pkg/proxy/conntrack/conntrack.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conntrack
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/pkg/proxy"
+	utilproxy "k8s.io/kubernetes/pkg/proxy/util"
+	"k8s.io/kubernetes/pkg/proxy/util/conntrack"
+	utilexec "k8s.io/utils/exec"
+)
+
+// CleanStaleConnections takes care of flushing stale conntrack entries for services and endpoints.
+func CleanStaleConnections(isIPv6 bool, exec utilexec.Interface, svcPortMap proxy.ServicePortMap,
```

Comment:
should be `CleanStaleConntrackEntries`; it's not cleaning _connections_.

Actually, I guess since the directory name is `conntrack` you shouldn't duplicate that in the function name, so just `CleanStaleEntries`, and the callers will say `conntrack.CleanStaleEntries(...)`